packages/template-chakra-storefront/src/ssr.js

```diff
@@ -26,7 +26,7 @@ const options = {
     defaultCacheTimeSeconds: 600,
 
     // The contents of the config file for the current environment
-    mobify: config.mobify,
+    mobify: config,
```

Comment:
For private slas client to work, we need to pass in other parts of the config, like the commerceAPI one.